### PR TITLE
Added LabMetadata Sync API endpoint

### PR DESCRIPTION
**Pull request by victorskl**
* Since HttpApi has limitation with async call, it is
  using boto to async invoke Lambda. Just quick simple
  solution. We shall revisit a better client experience
  with either AppSync GraphQL or WebSocket later.

### CODE REVIEW

**Comment by victorskl** [comment]
Ok, I will back port it with previous release (not so good practice). Just so that it can get into PROD, quicker. It is just independent and kinda _smaller-ish_ feature. So, should be fine. 

I will have one more minor patch fix about WTS tumor filtering, next. Then, I will push all together to PROD after then. FYI

**Review by williamputraintan — APPROVED** [review]
LGTM! Thanks!

**Comment by victorskl** [comment]
@reisingerf happy? no-happy?

File: data_portal/serializers.py [hunk 1]
```diff
@@ -163,22 +163,15 @@ def create(self, validated_data):
...
-        model = LIMSRow
-        fields = '__all__'
-
-
-class LabMetadataModelSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = LabMetadata
-        fields = '__all__'
+class LabMetadataSyncSerializer(serializers.Serializer):
+    sheets = serializers.ListField(default=['2020', '2021', '2022'])
```
Comment (reisingerf) [inline data_portal/serializers.py:167]: Should this include 2019 now?

**Review by reisingerf — APPROVED** [review]
Otherwise LGTM. Happy to look at this again in the future with hopefully better async support or performance improvements.

File: data_portal/serializers.py (hunk 1 above)
Comment (victorskl) [reply data_portal/serializers.py:167]: Right! Adding...

**Pull request merged**